src/status_im/contexts/wallet/sheets/select_account/view.cljs

```diff
@@ -25,12 +28,26 @@
         token                          (->> tokens
                                             (filter #(= (:symbol %) asset-symbol))
                                             first)
-        token-balance                  (get-in token [:balances-per-chain (:chain-id network) :balance])]
+        token-decimals                 (:decimals token)
```

Comment:
it would be great to have a util or data-store func that takes a `token` and returns `token-balance-display`, it will simplify the view logic, as other variables are currently only defined to support that